ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/EvmToolCommand.java

```diff
@@ -265,6 +265,10 @@ void setBytes(final String optionValue) {
       description = "display version info")
   boolean versionInfoRequested;
 
+  public Integer getRepeatCount() {
```

Comment:
```suggestion
  Integer getRepeatCount() {
```

nitpick: only needs to be package-private visibility